lms/djangoapps/experiments/utils.py

```diff
@@ -62,11 +61,27 @@
 # .. toggle_warnings: None
 # .. toggle_tickets: REVEM-118
 # .. toggle_status: supported
-DASHBOARD_INFO_FLAG = WaffleFlag(experiments_namespace,
-                                 u'add_dashboard_info',
-                                 flag_undefined_default=False)
+DASHBOARD_INFO_FLAG = WaffleFlag(experiments_namespace, u'add_dashboard_info')
```

Comment:
Removing the `flag_undefined_default` values on the above/unrelated WaffleFlags since they're being removed in https://github.com/edx/edx-platform/pull/24426/files#diff-68d4eb6dee8c8e94e217abf6af609d61L51 anyways and hoping to avoid merge/overwriting conflicts